pkg/arvo/app/landscape.hoon

```diff
@@ -41,68 +46,155 @@
   ^-  (quip card _this)
   |^
   ?+  mark  (on-poke:def mark vase)
+      %landscape-action
+    (landscape-action !<(action vase))
+  ::
       %handle-http-request
     =+  !<([id=@ta req=inbound-request:eyre] vase)
     :_  this
-    %+  give-simple-payload:app    id
-    %+  require-authorization:app  req
-    handle-http-request
+    %+  give-simple-payload:app  id
+    (handle-http-request req)
   ==
   ::
+  ++  landscape-action
+    |=  act=action
+    ^-  (quip card _this)
+    ?-  -.act
+        %serve-dir
+      :-  ~
+      this(serving (~(put by serving) url-base.act clay-base.act))
```

Comment:
Shouldn't this check for conflicts, and nack/crash if the binding is already in use?

Reply:
Ah good point.